parsers/objc.c

```diff
@@ -23,17 +23,6 @@
 #include "selectors.h"
 #include "vstring.h"
 
-/* To get rid of unused parameter warning in
- * -Wextra */
-#ifdef UNUSED
-#elif defined(__GNUC__)
-# define UNUSED(x) UNUSED_ ## x __attribute__((unused))
-#elif defined(__LCLINT__)
-# define UNUSED(x) /*@unused@*/ x
-#else
-# define UNUSED(x) x
-#endif
```

Comment:
The only possibly valuable thing I see here is the `__LCLINT__ ` part - should I add it to the global definition of UNUSED? Otherwise it tries to do the same as the global UNUSED.